client/extensions/woocommerce/store-sidebar/index.js

```diff
@@ -74,90 +64,120 @@ class StoreSidebar extends Component {
-						{ button.label }
-					</SidebarButton>
-				);
-			} );
-
-			// If this item has a parentSlug, only render it if 1) the parent is
-			// currently selected, 2) it is currently selected, or 3) any of its
-			// siblings are selected
-			if ( 'undefined' !== typeof item.parentSlug ) {
-				const links = [];
-				const parentItem = find( items, { slug: item.parentSlug } );
-				links.push( getLink( parentItem.path, site ) );
-
-				filter( items, { parentSlug: item.parentSlug } ).map( child => {
-					links.push( getLink( child.path, site ) );
-				} );
-
-				if ( ! this.isItemLinkSelected( links ) ) {
-					return null;
-				}
-			}
-
-			// If we reach this point, this is not a child item
-			// lets see if its children, if any, are selected
-			const childLinks = [];
-			filter( items, { parentSlug: item.slug } ).map( child => {
-				childLinks.push( getLink( child.path, site ) );
-			} );
-
-			const selected = this.isItemLinkSelected( itemLink ) || page.parentPath === item.path;
-			// Build the classnames for the item
-			const itemClasses = classNames( item.slug,
-				{
-					'has-selected-child': this.isItemLinkSelected( childLinks ),
-					'is-child-item': isChild,
-					'is-placeholder': isDisabled,
-					selected,
+	dashboard = () => {
+		const { site, siteSuffix, translate } = this.props;
+		const link = '/store' + siteSuffix;
+		const selected = this.isItemLinkSelected( link );
+		const classes = classNames( {
+			dashboard: true,
+			'is-placeholder': ! site,
+			selected,
+		} );
+
+		return (
+			<SidebarItem
+				className={ classes }
+				icon="house"
+				label={ translate( 'Dashboard' ) }
+				link={ link }
+				onNavigate={ this.onNavigate }
+				preloadSectionName={ 'dashboard' } />
+		);
+	}
+
+	products = () => {
+		const { site, siteSuffix, translate } = this.props;
+		const link = '/store/products' + siteSuffix;
+		const addLink = '/store/product' + siteSuffix;
+		const selected = this.isItemLinkSelected( [ link, addLink ] );
+		const classes = classNames( {
+			products: true,
+			'is-placeholder': ! site,
+			selected,
+		} );
+
+		return (
+			<SidebarItem
+				className={ classes }
+				icon="product"
+				label={ translate( 'Products' ) }
+				link={ link }
+				onNavigate={ this.onNavigate }
+				preloadSectionName={ 'products' } >
+				<SidebarButton disabled={ ! site } href={ addLink } >
+					{ translate( 'Add' ) }
+				</SidebarButton>
+			</SidebarItem>
+		);
+	}
+
+	orders = () => {
+		const { orders, site, siteSuffix, translate } = this.props;
+		const link = '/store/orders' + siteSuffix;
+		// We don't use the addLink yet, but this ensures the item is selected on single views
+		const addLink = '/store/order' + siteSuffix;
+		const selected = this.isItemLinkSelected( [ link, addLink ] );
+		const classes = classNames( {
+			orders: true,
+			'is-placeholder': ! site,
+			selected,
+		} );
+
+		return (
+			<SidebarItem
+				className={ classes }
+				icon="pages"
+				label={ translate( 'Orders' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **99** times:
`translate( 'Order' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).